persistence/schema/shared_sql.go

```diff
@@ -119,6 +121,8 @@ func Insert(
 							  height=EXCLUDED.height`,
 		tableName, actorSpecificParam,
 		actor.Address, actor.PublicKey, actor.StakedTokens, actorSpecificParamValue,
+		// DISCUSS_IN_THIS_COMMIT: Why did we change this?
```

Comment:
I don't have a good answer for this, revert?

Reply:
I reverted it and it broke the unit tests. Going to investigate

Reply:
Fixed: https://github.com/pokt-network/pocket/pull/165/commits/7148c17685b28f0734139447c19550712305ea2f